orchagent/muxorch.cpp

```diff
@@ -830,9 +836,90 @@ bool MuxOrch::isNeighborActive(IpAddress nbr, string alias)
         return ptr->isActive();
     }
 
+    string port;
+    if (!getMuxPort(mac, alias, port))
+    {
+        SWSS_LOG_INFO("Mux get port from FDB failed for '%s' mac '%s'",
+                       nbr.to_string().c_str(), mac.to_string().c_str());
+        return true;
+    }
+
+    if (!port.empty() && isMuxExists(port))
+    {
+        MuxCable* ptr = getMuxCable(port);
+        return ptr->isActive();
+    }
+
+    return true;
+}
+
+bool MuxOrch::getMuxPort(const MacAddress& mac, const string& alias, string& portName)
+{
+    portName = std::string();
+    Port rif, port;
+
+    if (!gPortsOrch->getPort(alias, rif))
+    {
+        SWSS_LOG_INFO("Interface '%s' not found in port table", alias.c_str());
+        return false;
+    }
+
+    if (rif.m_type != Port::VLAN)
+    {
+        SWSS_LOG_INFO("Interface type for '%s' is not Vlan, type %d", alias.c_str(), rif.m_type);
+        return false;
+    }
+
+    if (!gFdbOrch->getPort(mac, rif.m_vlan_info.vlan_id, port))
+    {
+        SWSS_LOG_NOTICE("FDB entry not found: Vlan %s, mac %s", alias.c_str(), mac.to_string().c_str());
```

Comment:
Instead of using "alias.c_str()" for VLAN, should it be "rif.m_vlan_info.vlan_id" instead?
Also, please check LOG severity setting here to reflect the true severity (see my previous question above)

Reply:
both would convey the same data - `Vlan1000 `vs `1000`. I think, this severity can be at NOTICE level if it cannot find FDB entry.

Reply:
In the previous OS that I worked on the IP interface name could be named different from the VLAN that is is mapped to.  Not sure if that is also applicable to SONiC...  If Not possible with SONiC it should be fine.

Reply:
IMO, its not applicable in Sonic. We have this naming convention of interface prefixed as "Vlan" and typically its configured as Vlan+id.

Reply:
Sure.  Change is not required.